### PR TITLE
chore: adding benchmark for `handle_request`

### PR DESCRIPTION
**Pull request by shashitnak**
**Summary:**  
_Briefly describe the changes made in this PR._

**Issue Reference(s):**  
Fixes #... _(Replace "..." with the issue number)_

**Build & Testing:**

- [ ] I ran `cargo test` successfully.
- [ ] I have run `./lint.sh --mode=fix` to fix all linting issues raised by `./lint.sh --mode=check`.

**Checklist:**

- [ ] I have added relevant unit & integration tests.
- [ ] I have updated the [documentation] accordingly.
- [ ] I have performed a self-review of my code.
- [ ] PR follows the naming convention of `<type>(<optional scope>): <title>`

[documentation]: https://github.com/tailcallhq/tailcallhq.github.io/tree/develop/docs

### CODE REVIEW

File: benches/handle_request_bench.rs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    let server_config = tokio_runtime
+        .block_on(ServerConfig::new(blueprint, endpoints))
+        .unwrap();
+    let server_config = Arc::new(server_config);
+
+    c.bench_function("test_handle_request", |b| {
+        let server_config = server_config.clone();
+        b.iter(|| {
+            let server_config = server_config.clone();
+            tokio_runtime.spawn(async move {
```
Comment (amitksingh1490) [inline benches/handle_request_bench.rs:30]:
```suggestion
            tokio_runtime.block_on(async move {
```

File: benches/handle_request_bench.rs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                let req = Request::builder()
+                    .method("POST")
+                    .uri("http://localhost:8000/graphql")
+                    .body(hyper::Body::from(
+                        json!({
+                            "query": "query { posts { title } }"
+                        })
+                        .to_string(),
```
Comment (amitksingh1490) [inline benches/handle_request_bench.rs:38]: move this out as static

File: benches/handle_request_bench.rs (hunk 1 above)
Comment (shashitnak) [reply benches/handle_request_bench.rs]: made this change

File: benches/handle_request_bench.rs (hunk 2 above)
Comment (shashitnak) [reply benches/handle_request_bench.rs]: made this change

**Review by amitksingh1490 — APPROVED** [review]

**Pull request merged**